### PR TITLE
redirect to home page

### PR DESCRIPTION
**Pull request by MohanChhabaria**
CDC logo redirects to home page when clicked (login required)

### CODE REVIEW

File: src/templates/student/base.html [hunk 1]
```diff
@@ -43,8 +43,14 @@
                      style="margin-left: auto; margin-bottom: 1px; margin-top: 1rem; margin-right: auto;">
                     <a class="thumbnail pull-left">
                         &nbsp &nbsp
-                        <img src="{% static '/img/cdclogo.png' %}" alt="IIT Jodhpur" class="logo"
+                        {% if user.is_authenticated %}
```
Comment (Tanuj22) [inline src/templates/student/base.html:46]: Why are you putting this checks?

File: src/templates/student/base.html [hunk 2]
```diff
@@ -43,8 +43,14 @@
...
+                        <img src="{% static '/img/cdclogo.png' %}" alt="IIT Jodhpur" class="logo",href="http://spc.iitj.ac.in/",
```
Comment (Tanuj22) [inline src/templates/student/base.html:47]: Do not use absolute url

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+[[source]]
```
Comment (Tanuj22) [inline Pipfile:1]: Remove this file

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Tanuj22) [inline .vscode/settings.json:1]: Remove this file

**Comment by Tanuj22** [comment]
The PR template is there for a reason. Use it

File: src/templates/student/base.html (hunk 1 above)
Comment (MohanChhabaria) [reply src/templates/student/base.html:46]: @Tanuj22  to check if the user is logged in or not

Comment (Tanuj22) [reply src/templates/student/base.html:46]: A student shouldn't be able to access this page if he is not logged in right?

Comment (MohanChhabaria) [reply src/templates/student/base.html:46]: Oh, i misinterpreted it .I thought that if user is logged in then only the logo should redirect to home page.
I will change it.

**Pull request merged**